src/strategies/StrategyTypes.ts

```diff
@@ -59,8 +59,9 @@ export type PrimitiveJSON = {
 }
 
 export type OrderJSON = {
-  primitives: PrimitiveJSON[]
   data?: string
+  orderIndex?: number
```

Comment:
is there a case where it would not be required? If it's always required, should remove the `?`